### ISSUE TITLE
improve initial question layout

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/gamma-viewer#3]
Try some 1 and two hops with predicates.   The nodes are frequently overlapped and the edges are usually unreadable until the nodes are moved around.

**Comment by maximusunc** [comment]
Is this for the aggregate graph view? Do you have a sample message for me to test with?

**Comment by cbizon** [comment]
No, for the question view.  Pretty much any of the questions in the notebook here: https://github.com/ranking-agent/QuestionRewrite/blob/master/documentation/QuestionAugmentationEdges.ipynb

So the original 1-hop looks like this:

![image](https://user-images.githubusercontent.com/10744350/80595174-4f13a580-89f2-11ea-942e-80441a3b69e4.png)
Note that the predicate is hidden behind n0

A little further down, a 3 hop looks like this:
![image](https://user-images.githubusercontent.com/10744350/80595278-75394580-89f2-11ea-8d68-a95285074198.png)
Besides the kind of odd layout, there are overlapping nodes.
